.github/workflows/feature_deploy.yml

```diff
@@ -0,0 +1,585 @@
+name: Feature environment deployment
+
+on:
+  push:
+    branches:
+      - feat/FN-72/deploy-app-code
+
+env:
+  environment: feature
+  credentials: ${{ secrets.AZURE_DIGITAL_DEV }}
+  from: latest
+  acr: ${{ secrets.ACR_REGISTRY_FEATURE }}
+  acr_username: ${{ secrets.ACR_USERNAME_FEATURE }}
+  acr_password: ${{ secrets.ACR_PASSWORD_FEATURE }}
+  # TODO: FN-72 NOTE: While we are using the dev GH environment, we can't use the RESOURCE_GROUP secret
+  # (It doesn't need to be a secret anyway.) So we define our resource group in an envioronment variable
+  # here and use that. The Azure region is the same so I've left that as using the secret.
+  resource_group: Digital-Feature
+
+jobs:
+  deployment-environment:
+    name: Environment setup
+    runs-on: [self-hosted, linux, deployment]
+    outputs:
+      environment: ${{ env.environment }}
+    steps:
+    - name: Initialise Environment
+      run: echo "Setting up ${{ env.environment }} environment"
+
+  #################### 1. PORTAL UI ####################
```

Comment:
Note that with all the apps deploying in parallel, we currently get "429 - Too many requests" errors. See https://github.com/UK-Export-Finance/dtfs2/actions/runs/6303025076/job/17111424514

I'll raise another card to address this. Is this an issue when deploying to `dev`? I can't see how it wouldn't be!